crates/router/src/lib.rs

```diff
@@ -99,6 +92,7 @@ pub fn mk_olap_app(
 /// # Panics
 ///
 ///  Unwrap used because without the value we can't start the server
+/// Starts the OLAP server with only OLAP services
```

Comment:
This description should be included in the beginning of the function documentation.

```rust
/// Description
///
/// # Errors/Panics/Safety/Examples (one or more sections)
///
/// ...
```

Also, move the `#[allow(...)]` annotations after the documentation comment, just before the function signature.